### PR TITLE
Upgrade to node 5.2.0 react 0.14.7

### PR DESCRIPTION
**Pull request by bradparker**

### CODE REVIEW

File: src/components/address/AddressLookup/__tests__/AddressLookup-test.js [hunk 1]
```diff
@@ -53,8 +53,8 @@ describe('AddressLookup', function() {
   it('accepts an existing address', function() {
     var element = TestUtils.renderIntoDocument(<AddressLookup address={ addressFindResult.address } />);
     var breakdown = findByClass(element, 'AddressBreakdown');
```
Comment (slwen) [inline src/components/address/AddressLookup/__tests__/AddressLookup-test.js]: Don't think this variable is used anymore?

File: src/components/address/AddressLookup/__tests__/AddressLookup-test.js [hunk 2]
```diff
@@ -62,15 +62,15 @@ describe('AddressLookup', function() {
   it('can be uniquely prefixed', function() {
     var element = TestUtils.renderIntoDocument(<AddressLookup prefix={ 'testPrefix-' } address={ addressFindResult.address } />);
     var breakdown = findByClass(element, 'AddressBreakdown');
```
Comment (slwen) [inline src/components/address/AddressLookup/__tests__/AddressLookup-test.js]: As above, unused variable now looks like

File: src/components/address/AddressLookup/__tests__/AddressLookup-test.js [hunk 3]
```diff
@@ -126,9 +126,9 @@ describe('AddressLookup', function() {
     expect(element.state.address).toBe(addressFindResult.address);
 
     var breakdown = findByClass(element, 'AddressBreakdown');
```
Comment (slwen) [inline src/components/address/AddressLookup/__tests__/AddressLookup-test.js]: Unused variable

File: src/components/address/AddressLookup/__tests__/AddressLookup-test.js (hunk 1 above)
Comment (bradparker) [inline src/components/address/AddressLookup/__tests__/AddressLookup-test.js]: Cheers, there might be a couple of these. Might see about eslint config to pick them up.

**Pull request merged**